config.js

```diff
@@ -7,7 +7,8 @@ const config = {
   checkRunName: 'Precaution',
   noIssuesResultTitle: 'No issues found',
   noIssuesResultSummary: 'There were no issues found.',
-  issuesFoundResultTitle: 'Issues found'
+  issuesFoundResultTitle: 'Issues found',
```

Comment:
Same change is in PR #145.  It's best to just create a commit with a dependency.

Reply:
I am not sure how to do that.

Reply:
I will now.